settings/eslint/eslintrc.js

```diff
@@ -0,0 +1,66 @@
+const commonParserOptions = {
+  ecmaFeatures: {
+    jsx: true,
+  },
+  ecmaVersion: '2020',
+  sourceType: 'module',
+};
+
+const commonRules = {
+  'prettier/prettier': [
```

Comment:
Let's put these in `.prettierrc.js` since it will have better support with IDEs

Reply:
Done. Thanks for suggestion.